recipes/gettsim/meta.yaml

```diff
@@ -0,0 +1,53 @@
+{% set name = "gettsim" %}
+{% set version = "0.6.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/gettsim-{{ version }}.tar.gz
+  sha256: 5c305d62b0c1d45344d659f0924af20f2f445d18bd15d3c368713aba453ef95a
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv
+  number: 0
+
+requirements:
+  host:
+    - python >=3.9
+    - setuptools >=45
+    - wheel
+    - setuptools-scm >=6.0
+    - pip
+  run:
+    - python >=3.9
+    - dags
+    - networkx
+    - numpy
+    - numpy_groupies
+    - pandas
+    - plotly
+    - pygments
+    - pygraphviz
+    - pytest
+    - pyyaml
+
+test:
+  imports:
+    - gettsim
+  commands:
+    - pip check
+  requires:
+    - pip
+
+about:
+  home: https://github.com/iza-institute-of-labor-economics/gettsim
+  summary: The German Taxes and Transfers SIMulator
+  license: AGPL-3.0-only
+  license_file: LICENSE
+
+extra:
+  recipe-maintainers:
+    - hmgaudecker
```

Comment:
Generally speaking, you should also be listing yourself as a maintainer since you're submitting the recipe:
```suggestion
    - hmgaudecker
    - timmens
```